src/ShipType.cpp

```diff
@@ -46,7 +46,7 @@ static bool ShipIsUnbuyable(const std::string &id)
 	return is_zero_exact(t.baseprice);
 }
 
-ShipType::ShipType(const Id &_id, const std::string &path) 
```

Comment:
More whitespace, there only appear to be a few real changes in this file but it's messy, and needs to be resolved against other master updates, due to this whitespace noise.